private-click-measurement.bs

```diff
@@ -159,50 +218,68 @@ Clients <dfn type=abstract-op>extract a six-bit decimal value</dfn> from a [=str
 
 </div>
 
-# Modern Triggering of Ad Click Attribution # {#moderntriggering}
+<div class=example id="valid-eight-bit-decimal-values">
 
-We envision a JavaScript API that is called on an ad click destination page as a modern means to trigger attribution at a [=conversion=]. This API call removes the necessity for third-party "pixels" which is great for ad click sources who do not want to be third party resources.
+The [=strings=] `"0"` and `"255"` are both [=eight-bit decimal values=], whereas `"398"` and `"!!11one"` are not.
 
-# Privacy Considerations # {#privacy}
+</div>
 
-The total entropy in ad click attribution HTTP requests is 12 bits (6+6), which means 4096 unique values can be managed for each [=ad click source-destination pair=].
+<div algorithm>
+
+Clients <dfn type=abstract-op>extract a eight-bit decimal value</dfn> from a [=string=] |string| by running these steps:
```

Comment:
This algorithm should be called "extract **an** eight-bit decimal value" (note the N).

Reply:
Will fix.